### DISCUSSION TITLE
Custom login

### DISCUSSION THREAD

**Discussion by haritz95** [filamentphp/filament#8568] [Help]
Hello everyone,

Is possible to edit the current login method? I want to offer the possibility to login with just a card number without any password but keeping the current option to do it with email and password.

Thank you

**Comment by RPotkowaVegasystems** [comment]
Try here: https://laraveldaily.com/post/filament-3-login-with-name-username-or-email. Maybe you can modify it to your needs.

**Reply by haritz95** [reply]
But how I could login without password? Tried that and works well but I would need to be able to login without password. If I'm not mistaken, filament uses password to login.
